Chummer/Utilities/frmDiceRoller.cs

```diff
@@ -285,14 +347,18 @@ private void cmdReroll_Click(object sender, EventArgs e)
                         && (nudDice.ValueAsInt & 1) == 0)))
             {
                 int intBubbleDieResult = GlobalOptions.RandomGenerator.NextD6ModuloBiasRemoved();
-                _lstResults.Add(new ListItem(intBubbleDieResult.ToString(GlobalOptions.InvariantCultureInfo),
-                    LanguageManager.GetString("String_BubbleDie") + strSpace + '(' + intBubbleDieResult.ToString(GlobalOptions.CultureInfo) + ')'));
+                ListViewItem lviCur=new ListViewItem(intBubbleDieResult.ToString(GlobalOptions.InvariantCultureInfo) + strSpace +
+                    LanguageManager.GetString("String_BubbleDie") + strSpace + '(' + intBubbleDieResult.ToString(GlobalOptions.CultureInfo) + ')');
```

Comment:
Why does this `ListViewItem` have only one argument in its constructor while the others have two? Seems unsafe.

Reply:
ListViewItems do not have a value any longer. I've cleaned up the second parameter on the other ones. I'm not entirly sure if the TryParse statement still works for the GlobalOptions.CultureInfo format. But we are not dealing with floating point numbers anyway.